components/header.tsx

```diff
@@ -24,6 +24,7 @@ export function Header({ locale = "en" }: Props) {
       <Select defaultValue={locale} onChange={onLangChange}>
         <option value="en">En</option>
         <option value="fr">Fr</option>
+        <option value="de">De</option>
```

Comment:
Nit: Having these sorted alphabetically makes it easier for users to find their language:
```suggestion
        <option value="de">De</option>
        <option value="en">En</option>
        <option value="fr">Fr</option>
```